python/opendht_cpp.pxd

```diff
@@ -187,7 +187,13 @@ cdef extern from "opendht/indexation/pht.h" namespace "dht::indexation":
     cdef cppclass Prefix:
         Prefix() except +
         Prefix(vector[uint8_t]) except +
+        Prefix(vector[uint8_t], size_t first) except +
+        Prefix(Prefix& p) except +
         string toString() const
+        string flagsToString() const
+        size_t size_
```

Comment:
Totally agreed with that, since flags_ and content_ are internal variable, user shouldn't be able to edit them directly.
As @aberaud point out, there is a risk of inconsistence ( ou just mistake if you [try to] move around those bits )